spec/dummy/app/themes/default/views/layouts/thredded.html.erb

```diff
@@ -23,5 +23,9 @@
 
     <%= javascript_include_tag 'thredded' %>
     <%= yield :thredded_javascript %>
+    <script>
+      thredded = (typeof thredded === 'undefined') ? new Thredded : thredded;
+      thredded.currentlyOnline.init();
```

Comment:
if `thredded` isn't defined, instantiate it and call the currently online method init.